docs/DafnyRef/Attributes.md

```diff
@@ -316,19 +316,60 @@ the functionality is already adequately described where
 refinement is described.
 -->
 
-### 22.2.10. `{:priority N}`
-Assign a positive priority 'N' to an implementation to control the order
-in which implementations are verified (default: N = 1).
+### 22.2.10. `{:priority}`
+`{:priority N}` assigns a positive priority 'N' to a method or function to control the order
+in which methods or functions are verified (default: N = 1).
+
+### 22.2.11. `{:rlimit}` {#sec-rlimit}
+
+`{:rlimit N}` limits the verifier resource usage to verify the method or function at `N * 1000`.
+This is the per-method equivalent of the command-line flag `/rlimit:N`.
+If using [`{:vcs_split_on_every_assert}`](#sec-vcs_split_on_every_assert) as well, the limit will be set for each assertion.
+
+To give orders of magnitude about resource usage, here is a list of examples indicating how many resources are used to verify each method:
+
+* 8K resource usage
+  ```dafny
+  method f() {
+    assert true;
+  }
+  ```
+* 10K resource usage using assertions that do not add assumptions:
+   ```dafny
+  method f() {
+    assert a: (a ==> b) <==> (!b ==> !a);
+    assert b: (a ==> b) <==> (!b ==> !a);
+    assert c: (a ==> b) <==> (!b ==> !a);
+    assert d: (a ==> b) <==> (!b ==> !a);
+  }
+  ```
+
+* 40K total resource usage using [`{:vcs_split_on_every_assert}`](#sec-vcs_split_on_every_assert)
+  ```dafny
+  method {:vcs_split_on_every_assert} f(a: bool, b: bool) {
+    assert a: (a ==> b) <==> (!b ==> !a);
+    assert b: (a ==> b) <==> (!b ==> !a);
+    assert c: (a ==> b) <==> (!b ==> !a);
+    assert d: (a ==> b) <==> (!b ==> !a);
+  }
+  ```
+*  37K total resource usage and fails out of resource.
```

Comment:
```suggestion
*  37K total resource usage and thus fails with `out of resource`.
```

Reply:
The error message does not mention "out of resources", only that assertions might not hold, so I can't put it like a quotation.

Reply:
It does for me:
```
filename.dfy(1,20): Verification out of resource (Impl$$_module.__default.f)

Dafny program verifier finished with 0 verified, 0 errors, 1 out of resource
```

Reply:
Oh I was mentioning the Dafny Language Server. We should fix this discrepancy.